src/components/chip-set/chip-set.tsx

```diff
@@ -628,6 +629,7 @@ export class ChipSet {
             identifier: chip.id,
             text: chip.text,
             icon: chip.icon,
+            image: chip.image,
```

Comment:
not sure why this broke the composite example 
<img width="425" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/c1124159-9aa5-4ffb-90ad-0deeb27d7d38">